acestep/constants.py

```
@@ -84,27 +84,31 @@
+# the HTTP fields land in a follow-up; Python callers using
+# ``GenerationParams(task_type="edit")`` work directly via inference.
 TASK_TYPES_BASE = ["text2music", "repaint", "cover", "cover-nofsq", "extract", "lego", "complete"]
 
 
 # ==============================================================================
 # Generation Mode Constants (UI-level modes that map to task types)
 # ==============================================================================
 
-# Default modes for turbo and SFT models (restricted set)
+# Default modes for turbo and SFT models (restricted set).  Edit is base-only
+# (paired CFG ≈ 4× decoder forwards/step is incompatible with turbo's CFG
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)